android/src/com/unciv/app/MultiplayerTurnCheckWorker.kt

```diff
@@ -32,6 +33,7 @@ class MultiplayerTurnCheckWorker(appContext: Context, workerParams: WorkerParame
 
     companion object {
         const val WORK_TAG = "UNCIV_MULTIPLAYER_TURN_CHECKER_WORKER"
+        const val LOG_TAG = "UNCIV_TURN_CHECK"
```

Comment:
This goes in the droid catlog? Verbatim? That log is getting hundreds of entries per µs, so some more is a good idea, but maximize readability anyway??

Reply:
No I mean where does that literal ***go***? Is it visible in the catlog? Is it custom to have all-caps there? The Log.i doc reads as if a class name is customary...